Pidaq/Lib/paralogging/logInit.py

```diff
@@ -0,0 +1,79 @@
+""" logInit.py
+
+This module contains helper functions for initializing loggers
+for the various paradigm research/testing rigs. The purpose of
+this module is to ensure log format consistency across all code """
+
+import logging
+import os
+from datetime import date
+
+
+def log_init(project, logger, formatStr=None, logFolderPath=None, logLevel=None):
+    """ Initialize the root logger for the specified system 
+    
+    Args:
+        project(str) - The name of the project (DTS, Stability, WindTunnel) 
+        
+        logger(Logger) - logger object, will generally be the root logger
+        
+        formatStr(str, optional) - The format log messages will be printed 
+            default - "%(levelname)s - %(asctime)s - %(name)s - %(message)s"
+            
+        logFolderPath(str, optional) - The path to the folder to place the log files,
+            (should end in /logs)
+            default - "cwd/logs"
+            
+        logLevel(str) - The log level of the root logger. Can be one of
+            [DEBUG, INFO, WARNING, ERROR, CRITICAL], any messages of lower
+            criticality will not be logged by the root logger.
+            default - DEBUG (All messages) 
+    """
+    if logLevel is None:
```

Comment:
Is there supposed to be an empty line above this? Reason I am wondering is because that is what I have been doing, and not sure what PEP8 says

Reply:
PEP-257 , docstring conventions:  "There's no blank line either before or after the docstring."